_includes/header.html

```diff
@@ -1,76 +1,80 @@
 		<span class="site-brand__name">Pa11y</span>
 	</a>
-	<div class="site-navigation" aria-label="Main Navigation">
-		<nav role="navigation">
-			<ul class="site-navigation__menu clearfix" tabindex="0" role="menubar" aria-controls="st1" >
 
-				<li class="site-navigation__item {% if page.url == '/' %}site-navigation__item--current{% endif %}" >
-					<a href="/" class="site-navigation__item-link">
+	<div class="site-nav" aria-label="Main Navigation">
+		<nav role="navigation">
+			<ul class="site-nav__menu clearfix" tabindex="0" role="menubar" aria-controls="st1" >
+				<li class="site-nav__item">
+					<a href="/" class="site-nav__link {% if page.url == '/' %}site-nav__link--current{% endif %}">
 						Home
 					</a>
 				</li>
 
-				<li class="site-navigation__item {% if page.url contains '/news/' %}site-navigation__item--current{% endif %}" >
-					<a href="/news/" class="site-navigation__item-link">
+				<li class="site-nav__item">
+					<a href="/news/" class="site-nav__link {% if page.url contains '/news/' %}site-nav__link--current{% endif %}">
 						News
 					</a>
 				</li>
 
-				<li class="site-navigation__item {% if page.url == '/tutorials/' %}site-navigation__item--current{% endif %}" >
-					<a href="/tutorials/" class="site-navigation__item-link">
+				<li class="site-nav__item">
+					<a href="/tutorials/" class="site-nav__link {% if page.url == '/tutorials/' %}site-nav__link--current{% endif %}">
 						Tutorials
 					</a>
 				</li>
 
-				<li class=" site-navigation__item {% if page.url == '/contributing/' %}site-navigation__item--current{% endif %} site-navigation__item--has-submenu" data-js-menu="true">
-					<a href="/contributing/" class="site-navigation__item-link" aria-haspopup="true">
+				<li class="site-nav__item site-nav__item--has-submenu" data-js-menu="true">
+					<button class="site-nav__link site-nav__link--button {% if page.url == '/contributing/' %}site-nav__link--current{% endif %}" aria-haspopup="true">
 						Contributing
-					</a>
-					<ul class="site-navigation__menu--sub-nav" aria-label="submenu" tabindex="-1"  >
+					</button>
+
+					<ul class="site-nav__menu site-nav__menu--sub" aria-label="submenu" tabindex="0">
+						<li class="site-nav__item site-nav__item--sub">
+							<a href="/contributing" tabindex="0" class="site-nav__link {% if page.url == '/contributing/' %}site-nav__link--current{% endif %}">
+								Overview
+							</a>
+						</li>
 
-						<li class="site-navigation__item  site-navigation__item--sub-nav  site-navigation__item {% if page.url == '/contributing/code-of-conduct/' %}site-navigation__item--current{% endif %}">
-							<a href="/contributing/code-of-conduct/" tabindex="-1" class="site-navigation__item-link">
+						<li class="site-nav__item site-nav__item--sub">
+							<a href="/contributing/code-of-conduct/" tabindex="0" class="site-nav__link {% if page.url == '/contributing/code-of-conduct/' %}site-nav__link--current{% endif %}">
```

Comment:
I think the tabindexes can be removed entirely. `a` is already focusable by default, so having `tabindex="0"` doesn't do anything. 

I don't really understand why the `tabindex="-1"` existed on these links in the first place 🤔I assume it must have been something for preventing people accidentally tabbing into them when the menu was closed? (??? or something)

Reply:
Yeah possibly....I wasn't sure either. I'll take those out then :-)